projects/canopy/src/lib/brand-icon/brand-icons.stories.ts

```diff
@@ -96,22 +95,6 @@ class SwatchBrandIconComponent implements OnChanges {
 
 const sizes = ['xxs', 'xs', 'sm', 'md', 'lg', 'xl', 'xxl'];
```

Comment:
Should we have this as an `enum` somewhere?

Reply:
I'm not sure, as it's just used to create the options for controls on the story, and not by the component itself. The `size` input on the component uses this type: 

```
export type BrandIconSize = 'xxs' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl';`
```

What do you think? I'd say maybe leave it as is for now as it's how was set before the upgrade and I don't think it would be currently useful outside this story. Happy to discuss though 🙂

Reply:
I think we said we were going to change those types to enums so that we can also use them better in the stories. @paul-request , what do you think?

Reply:
Yes, in order to reuse, rather than redefining in the story I suggested converting types to enums so that we can just export the enum from the component/inferface, and use it in CSF definition:
```
table: {
  type: {
    summary: Object.values(BrandIconSize),
  },
}

It seems like this type is already exported in `brand-icon.component.ts`, so we should convert to an enum and use as above IMO.